### PR TITLE
Add Folding@home SARS-CoV-2 RBD:ACE2 simulations

### PR DESCRIPTION
**Pull request by jchodera**
## Description
This PR adds a Folding@home simulation dataset for the SARS-CoV-2 RBD:ACE2 interface.

This should be ready to go pending reviews/feedback from @zhang-ivy @glass-w and MolSSI reviewers.

## Status
- [x] YAML file for each piece of data
- [x] Ready to go

### CODE REVIEW

File: data/simulations/FAH_SARS-CoV-2_ACE2_RBD.yml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    - 1R42
+trajectory: https://fah-public-data-covid19-antibodies.s3.us-east-2.amazonaws.com/vir-collaboration/SARS-CoV-2-ACE2-RBD/munged/solute/PROJ17311/run3-clone0.h5
+size: 2.1 TB
+length: 173.8 µs
+ensemble: NPT
+temperature: 310
+pressure: 1
+solvent: water
+salinity: 0
+forcefields:
```
Comment (zhang-ivy) [inline data/simulations/FAH_SARS-CoV-2_ACE2_RBD.yml:99]: Can you also add `GLYCAM_06j-1` as one of the forcefields used?

File: data/simulations/FAH_SARS-CoV-2_ACE2_RBD.yml [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    - 6m0j
+    - 1R42
+trajectory: https://fah-public-data-covid19-antibodies.s3.us-east-2.amazonaws.com/vir-collaboration/SARS-CoV-2-ACE2-RBD/munged/solute/PROJ17311/run3-clone0.h5
+size: 2.1 TB
+length: 173.8 µs
+ensemble: NPT
+temperature: 310
+pressure: 1
+solvent: water
+salinity: 0
```
Comment (zhang-ivy) [inline data/simulations/FAH_SARS-CoV-2_ACE2_RBD.yml:98]: 0.150 M NaCl

File: data/simulations/FAH_SARS-CoV-2_ACE2_RBD.yml [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+organization: Folding@home
+lab: Chodera lab
+institute: Memorial Sloan Kettering Cancer Center
+proteins:
+    - spike
+    - RBD
+    - ACE2
+structures:
+    - 6m0j
+    - 1R42
```
Comment (zhang-ivy) [inline data/simulations/FAH_SARS-CoV-2_ACE2_RBD.yml:90]: 1r42

File: data/simulations/FAH_SARS-CoV-2_ACE2_RBD.yml [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+  aws s3 --no-sign-request sync s3://fah-public-data-covid19-antibodies/vir-collaboration/SARS-CoV-2-ACE2-RBD/munged/solute/PROJ17311 .
+  ```
+
+  **Entire dataset:**
+  The raw Folding@home dataset is made available through the [AWS Open Data Registry](https://registry.opendata.aws/foldingathome-covid19/) and can be retrieved through the [AWS CLI](https://aws.amazon.com/cli/).
+  The dataset consists of a single project (`PROJ17311`) and has a `RUN*/CLONE*/result*` directory structure.
+  `RUNs` denote different mutants:
+  `RUN0` is RBD N439K;
+  `RUN1` is RBD K417V;
+  `RUN2` is N439K/K417K;
```
Comment (zhang-ivy) [inline data/simulations/FAH_SARS-CoV-2_ACE2_RBD.yml:61]: N439K/K417V

**Review by zhang-ivy — CHANGES_REQUESTED** [review]
Suggested some minor changes, otherwise looks good to me!

**Review by glass-w — COMMENTED** [review]
With @zhang-ivy 's suggestions it looks good to me.

File: data/simulations/FAH_SARS-CoV-2_ACE2_RBD.yml (hunk 3 above)
Comment (jchodera) [reply data/simulations/FAH_SARS-CoV-2_ACE2_RBD.yml:90]: This is how it appears in the [`data/structures/1R42.yml`](https://github.com/MolSSI/covid/blob/master/data/structures/1R42.yml) entry.

File: data/simulations/FAH_SARS-CoV-2_ACE2_RBD.yml (hunk 1 above)
Comment (jchodera) [reply data/simulations/FAH_SARS-CoV-2_ACE2_RBD.yml:99]: Done!

**Comment by jchodera** [comment]
This should be ready to go, @Lnaden !

**Review by zhang-ivy — APPROVED** [review]
Looks good to me

**Pull request merged**